kimchi/src/linearization.rs

```diff
@@ -222,8 +237,8 @@ pub fn constraints_expr<F: PrimeField + SquareRootField>(
     // flags.
     if cfg!(feature = "check_feature_flags") {
         if let Some(feature_flags) = feature_flags {
-            let (feature_flagged_expr, _) = constraints_expr(None, generic);
-            let feature_flagged_expr = feature_flagged_expr.apply_feature_flags(feature_flags);
+            let (feature_flagged_expr, _) = constraints_expr(cs, false, generic);
```

Comment:
Oh, I also added a flag here in order to pass these tests in [this PR](https://github.com/o1-labs/proof-systems/pull/1312)